servicetalk-grpc-api/src/main/java/io/servicetalk/grpc/api/GrpcServerBuilder.java

```diff
@@ -361,7 +442,10 @@ public final ServerContext listenAndAwait(GrpcBindableService<?, ?, ?>... servic
     protected abstract void doAppendHttpServiceFilter(Predicate<StreamingHttpRequest> predicate,
                                                       StreamingHttpServiceFilterFactory factory);
 
-    private void appendCatchAllFilterIfRequired() {
+    protected void appendCatchAllFilterIfRequired() {
```

Comment:
Can not comment there, but can we also deprecate `executor/ioExecutor/bufferAllocator` top level methods? Server builder doesn't have a problem like multi-address client. We can take execution context from HttpServer.

Also not sure if we really need `GrpcExecutionStrategy` for now. Maybe we can deprecate that one as well and add later when we have a need? @bondolo wdyt?

Reply:
I'm fine to do that in a follow-up